flask_seasurf.py

```diff
@@ -104,6 +104,7 @@ class SeaSurf(object):
     def __init__(self, app=None):
         self._exempt_views = set()
         self._include_views = set()
+        self._exempt_urls = tuple()
```

Comment:
I chose `tuple` is because coming urls should be checked via `str.startswith()`. Param needs to be `tuple` type. Do you prefer `tuple` or `set`?